provisioner/qradar.yml

```diff
@@ -0,0 +1,95 @@
+---
+###############################################################################
+# NOTE: YOU MUST UPDATE THIS EVERY TIME WE UPDATE THE QRADAR AMI
+#
+# This is a limitation of the QRadar installation setup, it is not made to
+# handle ephemeral ip addresses on instance spin up so we have to do post
+# deploy fixes
+#
```

Comment:
@liquidat I did explain in the comment you're referencing what need updating, is there something I'm missing? And no, the AMI creation is not documented at at this time, I need to though because it was a giant pain in the ass.

Reply:
Ah, ok, then we just do that in a later step. Thanks.